src/SFA.DAS.ApplyService.Application/Organisations/UpdateOrganisation/UpdateOrganisationHandler.cs

```diff
@@ -39,5 +47,25 @@ private async Task<Organisation> UpdateOrganisationIfExists(UpdateOrganisationRe
 
             return null;
         }
+
+        private async Task<OrganisationAddresses> UpdateOrganisationAddressesIfExists(Organisation request)
+        {
```

Comment:
DRY

Reply:
UpdateOrganisationHandler is deleted, it is not needed